external/webserver/http_client.c

```diff
@@ -210,6 +216,34 @@ static int process_trailer_header(char *buf, unsigned int buf_len, struct http_r
 	return 1;
 }
 
+static void parse_keep_alive_header(struct http_client_t *client, char *value)
+{
+	char *token = NULL;
+	char str_keep_alive[HTTP_CONF_MAX_VALUE_LENGTH] = { 0, };
+
+	strncpy(str_keep_alive, value, HTTP_CONF_MAX_VALUE_LENGTH);
```

Comment:
How about moving this line after line 226?
Then we don't need to copy if `client->keep_alive_header_flag` is not equal to `1`.

Reply:
done